plugins/plugin-chart-echarts/src/Timeseries/controlPanel.tsx

```diff
@@ -338,6 +339,18 @@ const config: ControlPanelConfig = {
             },
           },
         ],
+        [
+          {
+            name: 'xAxisLabelRotation',
+            config: {
+              type: 'TextControl',
+              label: t('Rotate x axis label'),
+              default: xAxisLabelRotation,
+              renderTrigger: true,
+              description: t('Rotate x axis label value between -90 and 90'),
```

Comment:
Let's add a number validator here. Also we should mention that it's degrees, something like
```
              description: t('Rotation of x axis label in degrees'),
```